ada_feeding/ada_feeding/behaviors/acquisition/compute_action_constraints.py

```diff
@@ -122,6 +122,7 @@ def setup(self, **kwargs):
     @override
     def update(self) -> py_trees.common.Status:
         # Docstring copied from @override
+        self.logger.info(f"{self.name} [{self.__class__.__name__}::update()]")
```

Comment:
I would recommend moving these prints to "debug" instead of "info" if you want to put them in, but this info should already be printed by the DebugVisitor when log_level is set to debug, so perhaps you should remove.

Reply:
Ah, I fully missed the Debug Visitor and the new launchfile arg for log-level. Thanks for pointing it out!